platforms/chibios/converters/promicro_to_elite_pi/_pin_defs.h

```diff
@@ -32,5 +32,5 @@
 #define B6 21U
 
 // LEDs
-#define D5 12U
-#define B0 13U
+#define D5 10U // back pad
+#define B0 11U // back pad
```

Comment:
I'm not sold on this, because I understand that it may break boards that have already been built using this converter, but for consistency, I also changed these two mappings to the pads on the back of the elite_pi

Reply:
Without this change, 1: using an elite_pi in a pcb designed for promicro is not consistent with using a stemcell in a pcb designed for promicro (which is annoying but not a huge deal on it's own). 2: D5 connects to one bottom pin if you design for elite_c but connects to a different bottom pin if you design for promicro and then use the promicro_to_elite_pi converter. For these two reasons combined, I think it's better to make this change than it is not to